scss/styles.scss

```diff
@@ -2,9 +2,11 @@
   align-items: center;
   background-color: $white-primary;
   display: flex;
-  height: 768px;
-  justify-content: center;
-  width: 1366px;
+  flex-direction: column;
+  height: $total-height;
+  justify-content: flex-start;
```

Comment:
`flex-start` is the default value for `justify-content` so it can be removed without a problem